### DISCUSSION TITLE
Any experience from building RMA Shims from source?

### DISCUSSION THREAD

**Discussion by ehunt34** [3kh0/ext-remover#449] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So, bad news, google drive mirror is down because k12sysadmin found it, and auto-sh1mmer failed, twice. (both on my Laptop and Steam Deck). I was wondering if you could clone the ChromiumOS repo and follow the instructions to build a RMA Shim (not sh1mmer itself, but what sh1mmer injects into). I know there are proprietary components to Shims that aren't built, but I assume you can extract them from one of the Lenovo Shims and slot them in, provided they aren't architecture based (as in able to run on x86, x86_64, and ARM without recompiling). Anyone know if this would work?

**Comment by Aka-but-nice** [comment]
Alr, sure I do, hit me up on discord

**Reply by ehunt34** [reply]
Bold of you to assume I would get that information from you of all people. You're likely lying anyway because you've been pissing your pants over "a new exploit" when it is literally the same damn thing other people have found.

**Reply by ehunt34** [reply]
All you made was one fucking website.

**Reply by Aka-but-nice** [reply]
No, I have made many sites, including <a href="https://abc-cloaker.glitch.me">This,</a> <a href="https://spot-maze-chinchilla.glitch.me">This</a> and <a href="https://passly.glitch.me">this.</a>  Now stfu, and grow up, if you didn't want help, you should've not said anything.

**Reply by Aka-but-nice** [reply]
What is the point of coming back, if each time you're gonna act more retarded.

**Reply by 3kh0** [reply]
@ehunt34 shut up or get blocked

**Comment by piplup7575** [comment, marked as answer]
[https://obelous.dev/sh1mmer](https://obelous.dev/sh1mmer) 🤓

**Reply by Aka-but-nice** [reply]
Well, at least he got his answer lmao
